courses/free-courses-hi.md

```diff
@@ -307,6 +307,7 @@
 
 ### HTML and CSS
 
+* [Complete Web Dev using mern stack Love Babbar](https://www.youtube.com/watch?v=Vi9bxu-M-ag&list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD&pp=iAQB) (Love Babbar)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
I have mentioned the playlist link and the creator name that is "Love babbar"

Reply:
thiis is the playlist: https://www.youtube.com/playlist?list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD

Reply:
Changed and saved the new link

Reply:
Changes done. - Love babbar